packages/widgets/src/lib/components/InferenceWidget/widgets/ConversationalWidget/ConversationalWidget.svelte

```diff
@@ -13,7 +13,7 @@
 		AddedToken,
 	} from "@huggingface/tasks";
 	import { SPECIAL_TOKENS_ATTRIBUTES } from "@huggingface/tasks";
-	import { HfInference } from "@huggingface/inference";
+	import { HfInference, type Options } from "@huggingface/inference";
```

Comment:
```suggestion
	import { HfInference } from "@huggingface/inference";
	import type { Options } from "@huggingface/inference";
```
unless this is intended?